test/test-applications/calltarget/CallTargetNativeTest/Program.cs

```diff
@@ -0,0 +1,1085 @@
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace CallTargetNativeTest
+{
+    class Program
+    {
+        private static MemoryStream mStream = new MemoryStream();
+        private static StreamWriter sWriter = new StreamWriter(mStream);
+
+        static void Main(string[] args)
```

Comment:
Would this be a suitable test to run as part of our integration tests to make sure that we do not crash when doing this wide variety of CallTarget modification?

Reply:
If this is something we run in CI, we'll need to change the `net471` TFM to `net461` for our integration test infrastructure to more easily find it

Reply:
Yes, I think it would be great include this as part of some pipeline.

Reply:
I changed the TFM to `net461` in https://github.com/DataDog/dd-trace-dotnet/pull/1007/commits/04ae61a340872f36f5e93929e69b26305bc991dd